### PR TITLE
Few fixes for phazor

### PR DESCRIPTION
**Pull request by RSKYS**

### CODE REVIEW

File: src/phazor/phazor.c [hunk 1]
```diff
@@ -1370,9 +1370,8 @@ static int pipe_update(struct spa_loop *loop, bool async, uint32_t seq,
                      const void *_data, size_t size, void *user_data){
 
         pw_stream_disconnect(global_stream);
-        pipe_connect(loop, async, seq, _data, size, user_data);
+        return pipe_connect(loop, async, seq, _data, size, user_data);
         printf("UPDATE SAMPLERATE PIPE TO %d\n", pipe_set_samplerate);
```
Comment (Taiko2k) [inline src/phazor/phazor.c]: This line is unreachable now?

File: src/phazor/phazor.c [hunk 2]
```diff
@@ -1421,7 +1420,6 @@ void connect_pulse() {
 
     //printf("ph: Connect device\n");
 
-    ma_context_config c_config = ma_context_config_init();
```
Comment (Taiko2k) [inline src/phazor/phazor.c:1424]: Can you explain this one to me? Is it not needed?

Comment (RSKYS) [reply src/phazor/phazor.c:1424]: Seems it already had initiate value and therefore, it works fine without it

File: src/phazor/phazor.c (hunk 1 above)
Comment (RSKYS) [reply src/phazor/phazor.c]: Since it's int function, it should return value there, that's why I returned last function to ignore possible structure warn.

Comment (Taiko2k) [reply src/phazor/phazor.c]: But the printf statement is pointless there now

File: src/phazor/phazor.c (hunk 2 above)
Comment (Taiko2k) [reply src/phazor/phazor.c:1424]: Maybe we would leave it there if theres no error

Comment (RSKYS) [reply src/phazor/phazor.c:1424]: Okay, I'll push it again, it leaves a warn but if you wanna be sure then oki doki :)

Comment (Taiko2k) [reply src/phazor/phazor.c:1424]: Whats the warning?

File: src/phazor/phazor.c (hunk 1 above)
Comment (RSKYS) [reply src/phazor/phazor.c]: Do you want me to write printf above it?

Comment (Taiko2k) [reply src/phazor/phazor.c]: Just remove it then

Comment (RSKYS) [reply src/phazor/phazor.c:1374]: I simply return 0 maybe

File: src/phazor/phazor.c (hunk 2 above)
Comment (RSKYS) [reply src/phazor/phazor.c:1424]:
```
src/phazor/phazor.c:1424:23: warning: variable ‘c_config’ set but not used [-Wunused-but-set-variable]
 1424 |     ma_context_config c_config = ma_context_config_init();
      |                       ^~~~~~~~
```

**Pull request merged**